src/core/credGrainView.test.js

```diff
@@ -0,0 +1,405 @@
+// @flow
+
+import {CredGrainView} from "./credGrainView";
+import * as GraphUtil from "./credrank/testUtils";
+import * as LedgerUtil from "./ledger/testUtils";
+import {g} from "./ledger/testUtils";
+import * as uuid from "../util/uuid";
+
+describe("core/credGrainView", () => {
```

Comment:
please add a test for the case where the graph has a participant that's missing in the ledger

Reply:
accepted